sdk/core/core-tracing/src/utils/cache.ts

```diff
@@ -4,49 +4,25 @@
 import { Tracer } from "../interfaces";
 import { getGlobalObject } from "./global";
 
-// V1 = OpenTelemetry 0.1
-// V2 = OpenTelemetry 0.2
-// V3 = OpenTelemetry 0.6.1
-// V4 = OpenTelemetry 1.0.0-rc.0
-const GLOBAL_TRACER_VERSION = 4;
-// preview5 shipped with @azure/core-tracing.tracerCache
-// and didn't have smart detection for collisions
-const GLOBAL_TRACER_SYMBOL = Symbol.for("@azure/core-tracing.tracerCache3");
+// tracerCache will be updated when a new incompatible version of OTel is
+// shipped in core-tracing.
+const GLOBAL_TRACER_SYMBOL = Symbol.for(`@azure/core-tracing.tracerCache4`);
```

Comment:
I think if we keep the simple numbers we should leave comments about what numbers correspond with which minimum OT package requirements, otherwise this will be annoying to dig out of git history someday.

Reply:
Done! I added tracerCache3 and tracerCache4 comments. For tracerCache 1 and 2 history is little muddier so wasn't easy to track down.